spaceR/spaceR/ViewModels/LaunchpadViewModel.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  LaunchpadViewModel.swift
+//  SpaceR
+//
+//  Created by Kieran Woodrow on 2022/03/21.
+//
+
+import Foundation
+
+class LaunchpadViewModel {
+    
+    private var allLaunchpads: [Launchpads]
+    
+    init() {
+        self.allLaunchpads = []
+    }
+    
+    func setAllLaunchpads(launchpads: [Launchpads]) {
+        allLaunchpads = launchpads
+    }
+    
+    func getAllRockets() -> [Launchpads] {
```

Comment:
This can be a computed property 😄 

`var rocketList: [Launchpads] { ...`

Please also consider the naming I used.

Reply:
cool will do